docs/getting_started.md

```diff
@@ -193,6 +193,7 @@ spec:
     services:
       cidrBlocks:
       - 100.64.0.0/13
+    serviceDomain: cluster.local            
```

Comment:
I believe this is the default service domain, was that not the case in your cluster?

Reply:
ah nvm, it should be part of the generated manifests now

Reply:
CI is saying there's a trailing space here, can you fix it please?